mpt_trie/src/lib.rs

```diff
@@ -24,8 +25,5 @@ pub mod trie_ops;
 pub mod trie_subsets;
 pub mod utils;
 
-#[cfg(feature = "trie_debug")]
```

Comment:
This shouldn't be changed IMO. Why did you remove the conditional flag?

Reply:
Since it's needed by `eth-trie-tools` I assumed that it was better to have it always present. Should I activate the feature in `eth-trie-tools` instead?

Reply:
Yeah we don't want to expose it by default. eth-trie-tools will just import the dependency with the feature on

Reply:
Always having it increases compile time and compiled code size